launch/cylinder_spawner.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
launch/cylinder_spawner.launch
<https://github.com/CentroEPiaggio/force-torque-sensor/pull/2#discussion_r48130478>
:

usefull to test how the FT Sensor works in simulation.

2015-12-21 10:54 GMT+01:00 Carlos J. Rosales <notifications@github.com>:

> In launch/cylinder_spawner.launch
> <https://github.com/CentroEPiaggio/force-torque-sensor/pull/2#discussion_r48130478>
> :
>
> > @@ -0,0 +1,5 @@
> > +<launch>
>
> Is this file necessary for the FT sensor, or is it just for testing the
> grasp? If it is not used in the FT sensor code, could you take it out of
> the PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CentroEPiaggio/force-torque-sensor/pull/2/files#r48130478>
> .
>